decide/src/main/java/org/example/Decide.java

```diff
@@ -12,8 +12,31 @@ public void decide() {
         System.out.println(decideHelper() ? "YES" : "NO");
     }
 
+    public Boolean[] getConditionMetVector() {
+        return new Boolean[] {
+                condition0(),
+                condition1(),
+                condition2(),
+                condition3(),
+                condition4(),
+                condition5(),
+                condition6(),
+                condition7(),
+                condition8(),
+                condition9(),
+                condition10(),
+                condition11(),
+                condition12(),
+                condition13(),
+                condition14()
+        };
+    }
+
     public boolean decideHelper() {
         // TODO do stuff here
+        // Create a list of booleans, one for each LIC
```

Comment:
Maybe this comment should be in the getConditionMetVector method

Reply:
That's a good point, I actually forgot that was there, should probably remove the "todo" too hehe